src/test/java/com/microsoft/sqlserver/testframework/AbstractTest.java

```diff
@@ -270,23 +272,32 @@ public static void invokeLogging() {
         try {
             if (Constants.LOGGING_HANDLER_CONSOLE.equalsIgnoreCase(loggingHandler)) {
                 handler = new ConsoleHandler();
+                handler.setFormatter(new SimpleFormatter());
             } else if (Constants.LOGGING_HANDLER_FILE.equalsIgnoreCase(loggingHandler)) {
                 handler = new FileHandler(Constants.DEFAULT_DRIVER_LOG);
+                handler.setFormatter(new SimpleFormatter());
                 System.out.println("Look for Driver.log file in your classpath for detail logs");
+            } else if (Constants.LOGGING_HANDLER_STREAM.equalsIgnoreCase(loggingHandler)) {
+                handler = new StreamHandler(new PrintStream(Constants.LOGGING_STREAM), new SimpleFormatter());
             }
 
             if (handler != null) {
-                handler.setFormatter(new SimpleFormatter());
                 handler.setLevel(Level.FINEST);
                 Logger.getLogger(Constants.MSSQL_JDBC_LOGGING_HANDLER).addHandler(handler);
             }
-            // By default, Loggers also send their output to their parent logger.
-            // Typically the root Logger is configured with a set of Handlers that essentially act as default handlers
-            // for all loggers.
+
+            /*
+             * By default, Loggers also send their output to their parent logger. Typically the root Logger is
+             * configured with a set of Handlers that essentially act as default handlers for all loggers.
+             */
             Logger logger = Logger.getLogger(Constants.MSSQL_JDBC_PACKAGE);
             logger.setLevel(Level.FINEST);
+
+            // enable activity trace
+            TestUtils.setActivityTraceOn();
```

Comment:
Enabling Activity Trace for all tests could impact ActivityID test added in #1020 . Please update PR with all latest dev changes to verify.

Reply:
merged and tested, thx